public/app/features/dashboard-scene/scene/row-actions/RowActions.tsx

```diff
@@ -25,31 +18,6 @@ import { RowOptionsButton } from './RowOptionsButton';
 export interface RowActionsState extends SceneObjectState {}
 
 export class RowActions extends SceneObjectBase<RowActionsState> {
```

Comment:
An interesting thing i found when testing is that repeated row has row actions available. This means that the repeated row can be repeated further down. It was possible in the previous arch as well, tho it resulted in no effect. I think we should just disable the repeated row's RowActions, otherwise it's confusing and the results are confusing. WDYT?

Reply:
yes, your rigtht, it should not have any options action even

Reply:
@torkelo - i've just pushed a commit that handles that